test/test_qat.py

```diff
@@ -123,17 +122,21 @@ def ref_op(x):
             else:
                 ref_op = compose([conv_op, bn_op, relu_op])
 
-            result_ref = ref_op(input)
-            result_actual = qat_op(input)
-            self.assertEqual(result_ref, result_actual)
+            input_clone = input.clone().detach_().requires_grad_()
+            for i in range(2):
```

Comment:
This looks like a typo. Do you need the for loop here?

Reply:
yeah, I had some error previously when I run the op with forward and backward twice